package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@interviewdojo/algorithms",
+  "version": "0.0.0",
+  "description": "interview prep material",
+  "scripts": {
+    "test": "jest ./src --coverage",
+    "lint": "eslint ./src --ext ts,js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/InterviewDojo/Algorithms.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
```

Comment:
never done this myself, but did we need to do anything for the copyright and licensing of the project?

Reply:
https://github.com/git/git-scm.com/blob/main/MIT-LICENSE.txt

no liability